### ISSUE TITLE
The shape of the logits

### ISSUE THREAD

**Issue by neverUseThisName** [haofanwang/video-swin-transformer-pytorch#7]
the output 'logits' are of shape (1,768,8,7,7), but it should be (batch, num_class). How to adapt the code to classify videos?

**Comment by TheEighthDay** [comment]
> 

The fc layer is defined in (https://github.com/SwinTransformer/Video-Swin-Transformer/tree/master/mmaction/models/recognizers)/base.py   from the official implementation.

**Comment by hankyuwon** [comment]
I'm not completely sure, but in my opinion, you can perform video classification by applying softmax or sigmoid.
